demosplan/DemosPlanCoreBundle/Logic/Document/ParagraphHandler.php

```diff
@@ -10,20 +10,28 @@
 
 namespace demosplan\DemosPlanCoreBundle\Logic\Document;
 
-use DemosEurope\DemosplanAddon\Contracts\MessageBagInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\ElementsInterface;
+use DemosEurope\DemosplanAddon\Contracts\Entities\ProcedureInterface;
+use DemosEurope\DemosplanAddon\Contracts\Handler\ParagraphHandlerInterface;
 use demosplan\DemosPlanCoreBundle\Logic\CoreHandler;
 use demosplan\DemosPlanCoreBundle\Logic\FlashMessageHandler;
+use demosplan\DemosPlanCoreBundle\Logic\MessageBag;
+use demosplan\DemosPlanCoreBundle\Repository\ParagraphRepository;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
-class ParagraphHandler extends CoreHandler
+class ParagraphHandler extends CoreHandler implements ParagraphHandlerInterface
 {
     /** @var ParagraphService */
     protected $service;
```

Comment:
I guess this can be removed, as the ParagraphService is used in the contructor. I checked the ParagraphHandler and $this->service is not used, so it can be safely deleted. it is not a blocker thu

Reply:
done in f2edd5fc69cd6927f1bb2a358c130bd31409d56d